lib/ModelMixins/GeojsonMixin.ts

```diff
@@ -107,6 +107,18 @@ function GeoJsonMixin<T extends Constructor<Model<GeoJsonTraits>>>(Base: T) {
   abstract class GeoJsonMixin extends DiscretelyTimeVaryingMixin(
     MappableMixin(UrlMixin(Base))
   ) {
+    constructor(...args: any[]) {
+      super(...args);
+      if (this.strata.get(GeoJsonStratum.stratumName) === undefined) {
+        runInAction(() => {
+          this.strata.set(
+            GeoJsonStratum.stratumName,
+            GeoJsonStratum.load(this)
+          );
+        });
+      }
```

Comment:
Any reason to move this to constructor from `forceLoadMetadata`?

Reply:
I moved it because it doesn't load anything (there is no async call) - it is only used to set `rectangle`

Reply:
https://github.com/TerriaJS/terriajs/blob/90931527d585fda06e87152f94235045802259ff/lib/Core/AsyncLoader.ts#L9

Reply:
We should only use AsyncLoaders if there is an async call